Cargo.toml

```diff
@@ -22,6 +22,11 @@ default = ["std", "impl-cgmath"]
 serialization = ["serde", "serde_derive"]
 std = []
 impl-cgmath = ["cgmath"]
+impl-nalgebra = ["nalgebra"]
+
+[dependencies.nalgebra]
+version = "0.14"
```

Comment:
The latest version is 0.16. I saw in the commits diff that you moved away from 0.16 to 0.14. Why?

Reply:
I moved to 0.14 because that is what the current version of ggez depends on, and using 0.16 prevented me from building splines and ggez together.

I think "^0.14" might be more appropriate, or I could in principal determine the lowest version where this change works.

Reply:
Oh, I see. What changes from 0.14 that makes it impossible to use 0.15 and 0.16?

Reply:
I don't really understand what makes it not work.
On 1.6 I get an error starting that "the trait bound `ggez::nalgebra::Point<f32, ggez::nalgebra::U2>: splines::Interpolate` is not satisfied".
The error is:
"the trait `splines::Interpolate` is not implemented for `ggez::nalgebra::Point<f32, ggez::nalgebra::U2>".
I don't see any change between the versions in how Point is defined, or any of the types it is based on, but somehow the implementation does not apply to the Point type in 0.15 or 0.16.

Reply:
I'm missing something fundamental here. In this case, splines and ggez are requiring different specific versions of nalgebra, and there should be no version that satisfies both constraints (0.14 and 0.16). This is apparently okay, as it will still try to build. However, its like 0.16 Points are different from 0.14 Points, and Interpolate is only defined on 0.16 Points? Is this how Rust works?

Reply:
This is tricky, as the typical way to deal with that would be to support 0.16 and have ggez to update. Maybe @icefox has an idea?

Reply:
Icefox does not want to update ggez, as it re-exports nalgebra which makes an update an interface change and requires a version bump. They mentioned a plan to move to mint as a common math interface, which might eventually be the answer here as well.
I can also make this dependency allow 0.14, 0.15, and 0.16, which all appear to work. 
I haven't tested that this actually compiles ggez with 0.14, but it seems like it should?

Reply:
If it works for all the three versions, that’s perfect!

Reply:
Ok, I fixed it in a commit of mine.